index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Calculator</title>
+    <!-- from https://www.bootstrapcdn.com/ -->
+    <!--- see also https://getbootstrap.com/docs/4.0/getting-started/download/ -->
+    <link
+        href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css"
+        rel="stylesheet"
+        integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk"
+        crossorigin="anonymous"
+    >
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <script>
+        /**
+         * `calculator` is a variable that is an object. The {} brackets denote the variable is an object.
+         * `add`, `subtract` and `multiply` are functions. They are also properties of the `calculator` object.
+         */
+        calculator = { 
+            add: function () {
+                document.forms.inputs.result.value = parseInt(document.forms.inputs.number1.value)+ parseInt(document.forms.inputs.number2.value)
+            },
+            subtract: function () {
+                document.forms.inputs.result.value = document.forms.inputs.number1.value - document.forms.inputs.number2.value
+            },
+            multiply: function () {
+                document.forms.inputs.result.value = document.forms.inputs.number1.value * document.forms.inputs.number2.value
+            },
+            divide: function () {
+                document.forms.inputs.result.value = (document.forms.inputs.number1.value) / (document.forms.inputs.number2.value)
+            },
+        }
+    </script>
+    <style>
+        body {
+            padding: 30px;
+            background-color:darkblue 
+        }
+        .result {
+            padding-top: 20px;
+        }
+    </style>
```

Comment:
Try to get in the habit of putting your css (styling of the page) in a separate styles.css file.